tests/rest/test_middleware.py

```diff
@@ -23,37 +23,37 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-@patch("taipy.rest.api.middlewares._middleware._using_enterprise")
 @patch("taipy.rest.api.middlewares._middleware._enterprise_middleware")
+@patch("taipy.common._check_dependencies.EnterpriseEdition._is_installed")
```

Comment:
```suggestion
@patch("taipy.common._check_dependencies.EnterpriseEdition._is_installed")
@patch("taipy.rest.api.middlewares._middleware._enterprise_middleware")
```

The order matters.

Reply:
Oh yes it does. I thought it would be the same order... and did not check.
Now I know.
Thx